ci/test/pipeline.template.yml

```diff
@@ -541,6 +541,14 @@ steps:
 
   # Fast tests closer to the end, doesn't matter as much if they have to wait
   # for an agent
+  - id: vet-check
+    label: Vet Check
+    command: bin/ci-builder run stable cargo vet check
+    timeout_in_minutes: 5
+    agents:
+      queue: linux
```

Comment:
If you use linux-x86_64 it should choose the cheaper agents, but probably doesn't matter much. (Might even cost more if the agent has to be spun up first.)